src/browser/karma.conf.js

```diff
@@ -47,7 +47,7 @@ module.exports = function(config) {
       // suppress annoying 404 warnings for resources, images, etc.
       { pattern: 'dist/dev/assets/**/*', watched: false, included: false, served: true },
 
-      'test-main.js'
+      'src/browser/test-main.js'
```

Comment:
Currently `test-main.js` gets copied over to `dist/dev/test-main.js`.
Question: Should it point to the copied `test-main.js` in `dist/dev` or the source `test-main.js` in `src/browser`?